### ISSUE TITLE
TaskCategory: Add Category

### ISSUE THREAD

**Issue by mfuerni** [jguertl/SWWM#5]
Add a new Category to the list.

The user taps on the "Add Category" button in the category list, enters the name of the category end taps OK. Then the refreshed list of categories is displayed.

Requires: #4

**Comment by mfuerni** [comment]
Merged into develop with 59d7190151a3ff920245e658b968f85c3a36437d
